### ISSUE TITLE
Available loss functions

### ISSUE THREAD

**Issue by lorenzomammana** [orobix/quadra#11]
### Description & Motivation

Right now the amount of available standard losses is very limited, for example for classification we have only bce (which is actually bce with logits), it would be useful to add some more standard losses from nn to simplify switching from one to another.
